src/components/Editor.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef } from "react";
+import EmailEditor, { UnlayerOptions, User } from "react-email-editor";
+import mergeTags from "../external/merge.tags";
+
+export interface EditorProps {
+  design?: any;
```

Comment:
What do you think about using a better type than `any` here? `react-email-editor` should define it.